dafni_cli/model_metadata.py

```diff
@@ -0,0 +1,200 @@
+from typing import Optional
+
+from dafni_cli.consts import (
+    INPUT_TITLE_HEADER,
+    INPUT_TYPE_HEADER,
+    INPUT_MIN_HEADER,
+    INPUT_MAX_HEADER,
+    INPUT_DEFAULT_HEADER,
+    INPUT_DESCRIPTION_HEADER,
+    INPUT_TYPE_COLUMN_WIDTH,
+    INPUT_MIN_MAX_COLUMN_WIDTH,
+    INPUT_DESCRIPTION_LINE_WIDTH,
+    OUTPUT_NAME_HEADER,
+    OUTPUT_FORMAT_HEADER,
+    OUTPUT_SUMMARY_HEADER,
+    OUTPUT_FORMAT_COLUMN_WIDTH,
+    OUTPUT_SUMMARY_COLUMN_WIDTH,
+    TAB_SPACE,
+)
+
+
+def optional_column(
```

Comment:
these few functions here also look quite useful. Assuming we should be able to use these elsewhere? Might be worth locating in a more central location

Reply:
Just been playing around with the new get model command. The table you've managed to get setup looks really good!